json_serializable/lib/src/type_helpers/json_helper.dart

```diff
@@ -21,11 +21,15 @@ class JsonHelper extends TypeHelper {
   /// By default, JSON encoding in from `dart:convert` calls `toJson()` on
```

Comment:
nit: update this comment

Reply:
how?

Reply:
By noting that if `context.explicitToJson` is `true` these calls will be injected automatically.

Reply:
done